src/styles/dropdown.css

```diff
@@ -0,0 +1,40 @@
+.sub-nav li {
+  padding: 18px;
+  width: calc(100%-18px);
+  border-bottom: 1px solid rgba(0,0,0,0.1);
+  text-align: center;
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
+  line-height: 1;
+}
+
+.sub-nav a:hover {
+  color: #3d4351;
+}
+
+.home-nav-menu .flat-button {
+  color: #fff;
+  border-top-left-radius: 0px;
+  border-bottom-left-radius: 0px;
+  margin-right: 0px;
+}
+
+.home-nav-menu .trial-input {
+  border-color: rgba(61, 67, 87, 0.1);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```